### PR TITLE
Remove unused variables

### PR DESCRIPTION
**Pull request by anwar6953**
Perfection is reached not when there’s nothing left to add, but when there’s nothing left to remove.

### CODE REVIEW

**Comment by ajainarayanan** [comment]
@anwar6953 fileselect.js , status-ctrl.js in workflows can be removed. @elfenheart Can you check why we are not using runids in flowlet input-ctrl.js ?

Otherwise LGTM

**Comment by elfenheart** [comment]
@anwar6953 @ajainarayanan in pending metrics, there is no concept of runid. It is always an aggregation. This is the fix that Andreas created for system.queue.pending metrics for flowlet

**Comment by anwar6953** [comment]
It's strange then that we show outgoing metrics by runId, but pending metrics across runIds.
Can be quite confusing when looking at the values.

**Comment by elfenheart** [comment]
I think the fix should be to show Not Available or 0 for past runs. That information is only useful for the currently running flows.

**Comment by anwar6953** [comment]
Even if only showing information for the currently running flow, the pending is showing across all runs, while the outgoing is showing only for that run.

**Comment by elfenheart** [comment]
how else would you do it? We have the information for outgoing for specific run, but there is no run specific pending metrics

**Comment by anwar6953** [comment]
I'm not sure how I'd do it. But I definitely think it needs to be revisited. I'll remove runId from the code for now, and add it to Daily Tasks doc.

**Comment by anwar6953** [comment]
Ok, removed the remaining 4.

File: cdap-ui/app/directives/instance-control/instance-control.js [hunk 1]
```diff
@@ -30,9 +30,9 @@ angular.module(PKG.name + '.commons')
             method: 'PUT',
             _cdapPath: scope.basePath + '/instances',
             body: {'instances': scope.model.requested}
-          }).then(function success (response) {
+          }).then(function success () {
             scope.model.provisioned = scope.model.requested;
-          }, function error (err) {
+          }, function error () {
```
Comment (elfenheart) [inline cdap-ui/app/directives/instance-control/instance-control.js]: I think you can remove this entire error function. It's not doing anything

File: cdap-ui/app/features/workflows/controllers/tabs/runs/tabs/status-ctrl.js [hunk 2]
```diff
@@ -108,93 +106,6 @@ angular.module(PKG.name + '.feature.workflows')
...
-      sourceName: nodes[nodes.length - 2].name,
-      sourceType: nodes[nodes.length - 2].nodeType,
-      targetName: nodes[nodes.length - 1].name
-    });
-
-  }
-
-}
-
-/**
```
Comment (ajainarayanan) [inline cdap-ui/app/features/workflows/controllers/tabs/runs/tabs/status-ctrl.js]: This comment belongs to the function below it.. I don't believe we should remove this one?

File: cdap-ui/app/features/workflows/controllers/tabs/status-ctrl.js [hunk 3]
```diff
@@ -52,100 +49,13 @@ angular.module(PKG.name + '.feature.workflows')
...
-        sourceName: nodes[nodes.length - 2].name,
-        sourceType: nodes[nodes.length - 2].nodeType,
-        targetName: nodes[nodes.length - 1].name
-      });
-
-    }
-
-  }
-
-  /**
```
Comment (ajainarayanan) [inline cdap-ui/app/features/workflows/controllers/tabs/status-ctrl.js]: Same here too.

File: cdap-ui/app/features/workflows/controllers/tabs/runs/tabs/status-ctrl.js (hunk 2 above)
Comment (anwar6953) [inline cdap-ui/app/features/workflows/controllers/tabs/runs/tabs/status-ctrl.js]: Good point. I didn't realize that. Added back.

File: cdap-ui/app/directives/instance-control/instance-control.js (hunk 1 above)
Comment (anwar6953) [inline cdap-ui/app/directives/instance-control/instance-control.js]: removed.

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**